packages/frontend-api/src/Model/Complaint/ComplaintItemDataApiFactory.php

```diff
@@ -31,8 +34,13 @@ public function __construct(
      */
     public function createFromComplaintItemInput(OrderItem $orderItem, array $item): ComplaintItemData
     {
+        $product = $orderItem->getProduct();
+
         $complaintItemData = $this->complaintItemDataFactory->create();
         $complaintItemData->orderItem = $orderItem;
+        $complaintItemData->product = $product;
+        $complaintItemData->productName = $product->getName($this->domain->getLocale());
+        $complaintItemData->catnum = $product->getCatnum();
```

Comment:
`$orderItem->getCatnum()` should be used here as the product might be null

Reply:
fixed [fixup! complaints b2b customers support](https://github.com/shopsys/shopsys/pull/3295/commits/585c5b09e615c10c4351fc201d7abf686fa2e022)